sql/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -2498,7 +2498,7 @@ class SparkConnectPlanner(
     if (isCommand) {
       // Convert the results to Arrow.
       val schema = df.schema
-      val maxBatchSize = (SparkEnv.get.conf.get(CONNECT_GRPC_ARROW_MAX_BATCH_SIZE) * 0.7).toLong
+      val maxBatchSize = (ConnectCommon.CONNECT_GRPC_MAX_MESSAGE_SIZE * 0.7).toLong
```

Comment:
If this is more performance in most cases, I think we should change the default value of `CONNECT_GRPC_ARROW_MAX_BATCH_SIZE` instead?

Reply:
I did not want to change the value for `processAsArrowBatches`, but looking into it, I think changing the value there makes sense as well since there the ExecutePlanResponse also only contains the ArrowBatch. So increasing the limit here to 0.7 * MAX_MESSAGE_SIZE is fine as well. Let's change the config instead of only the value here then! Thanks!

Reply:
Changed.